webpack.config.js

```diff
@@ -189,6 +193,10 @@ function createConfig(isProdBuild, latestBuild) {
             "node_modules/@polymer/font-roboto-local/fonts/roboto/Roboto-Bold.ttf",
         },
       }),
+      new webpack.ProvidePlugin({
```

Comment:
Can you remove this.

Reply:
ah yes. Sorry I forgot about that

Reply:
Taking this out results in `$ not defined` errors in the console

Reply:
I re-ran both bootstrap and develop with the webpack removed and the same error persists

Reply:
where is it not defined?

Reply:
AFK but Im pretty sure its just where I use the jQuery prefix `$`. I can give you a more solid answer in an hour or so

Reply:
Also, Travis is failing because of the `$` even though the webpack plugin is still in the code